app/src/common/shared/com/igalia/wolvic/ui/widgets/SystemNotificationsManager.java

```diff
@@ -0,0 +1,82 @@
+package com.igalia.wolvic.ui.widgets;
+
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+
+import com.igalia.wolvic.R;
+import com.igalia.wolvic.ui.adapters.SystemNotification;
+import com.igalia.wolvic.utils.SystemUtils;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+
+public class SystemNotificationsManager {
+    private static final String LOGTAG = SystemUtils.createLogtag(SystemNotificationsManager.class);
+
+    private static final int NOTIFICATION_DURATION = 5000;
+
+    private static final SystemNotificationsManager mInstance = new SystemNotificationsManager();
+
+    // TODO the system notifications should be stored until the used has dismissed them
+    private final List<SystemNotification> mSystemNotifications = Collections.synchronizedList(new ArrayList<>());
+
+    private final Set<ChangeListener> mChangeListeners = new LinkedHashSet<>();
+
+    private SystemNotificationsManager() {
+        mSystemNotifications.add(new SystemNotification("TEST Notification title", "Notification body",
+                new SystemNotification.Action(SystemNotification.Action.OPEN_URL, null, "https://google.com", null),
+                Calendar.getInstance()));
+        mSystemNotifications.add(new SystemNotification("TEST Notification title 2", "Notification body 2",
+                new SystemNotification.Action(SystemNotification.Action.OPEN_URL, null, "https://reddit.com", null),
```

Comment:
All this seems debugging code?

Reply:
Yes, that's debugging code. I don't understand why it made it into the commit.